SSignalKit.podspec

```diff
@@ -0,0 +1,19 @@
+Pod::Spec.new do |s|
+
+  s.name         = "SSignalKit"
+  s.version      = "0.0.2"
+  s.summary      = "An experimental Rx- and RAC-3.0-inspired FRP framework"
+  s.homepage     = "https://github.com/PauloMigAlmeida/Signals"
```

Comment:
Wrong homepage URL.

Reply:
@kiopro, I think you got that all wrong..... I didn't try to get credit for anything I haven't done myself but given the fact that (up to this day) a Podspec file doesn't exist in this project and no other Pod (at that time) had been published to the public cocoapods repository then I needed to point potential developers to a page where they could find a more "up-to-date" reference to the project. 

Honestly speaking, it was part of my plan at that time to maintain a separate fork of it as I planned to take the project on a different direction.... however, it didn't catch on and I moved to other projects.

Regarding the ```social_media_url``` property, if you had paid closer attention to it you would've realised that since I couldn't find his email, it's only likely that I also couldn't find his social_media_url (a.k.a his profile), so in the case of the email I left it empty hoping that he could fill it out himself after merging the PR and  **of course**, the same behaviour was expected for the social_media_url since (at that time - not sure if that remains still) this wasn't an array.

Anyway, my whole point of view boils down to this:

1. It clear to all of us that no intention of getting undue credit was attempted 
2. It has never been part of the author's interest to make it available at cocoapods repo (for whatever the reason may be, I respect his choice)
3. People like you who go to random PRs and comment on a mere string that you couldn't be bothered to think why they were written in the first place isn't really helping anyone. I hope they can use this time to fix a bug instead (or at least make it easier for other people in the community to use it).

Regardless of that, @peter-iakovlev thanks for the project, that was a valuable contribution to the OS community and I commend you for doing that. 
@kiopro, thanks for pointing these things out, I should've closed this PR a long time ago and your comments served a purpose.

Best regards,